internal/goordinator/rule.go

```diff
@@ -136,32 +137,38 @@ func (r *Rule) Match(ctx context.Context, event *Event) (MatchResult, error) {
 	}
 }
 
-// TemplateActions templates the filter query of the rule for the specific event.
-func (r *Rule) TemplateActions(ctx context.Context, event *Event) ([]action.Runner, error) {
-	result := make([]action.Runner, 0, len(r.actions))
-
-	for _, actionDef := range r.actions {
-		templFun := func(in string) (string, error) {
-			templ, err := template.New("action").Parse(in)
-			if err != nil {
-				return "", err
-			}
+var templateFuncs = template.FuncMap{
+	"queryescape": url.QueryEscape,
+}
 
-			var out bytes.Buffer
+func templateFunc(event *Event) func(in string) (string, error) {
```

Comment:
I googled a bit and "to template" seems the wrong term for describing combining data + a template to generate an output. I'm not 100% certain though. :-)
Can you confirm that? 

Because multiple functions in different components have to be renamed, I don't want to do this change in this PR.
I created a separate PR (https://github.com/simplesurance/goordinator/pull/44), to replace the "template" with "render" everywhere. That could be merged after this PR.

Reply:
yeah, sounds good 👍